src/utils/test_util.ts

```diff
@@ -0,0 +1,82 @@
+import metadataRpc from '@polkadot/metadata/Metadata/v11/static';
+import { Metadata, TypeRegistry } from '@polkadot/types';
+import { getSpecTypes } from '@polkadot/types-known';
+
+/**
+ * Create a type registry for Kusama.
+ * Useful for creating types in order to facilitate testing.
+ */
+function createKusamaRegistry(): TypeRegistry {
+	const registry = new TypeRegistry();
+
+	registry.register(getSpecTypes(registry, 'Kusama', 'kusama', 2008));
+
+	registry.createType('ChainProperties', {
+		ss58Format: 2,
+		tokenDecimals: 12,
+		tokenSymbol: 'KSM',
+	});
+
+	registry.setMetadata(new Metadata(registry, metadataRpc));
+
+	return registry;
+}
+
+export const kusamaRegistry = createKusamaRegistry();
+
+/**
+ * An 'at' object, which has not been sanitized by `sanitizeNumbers`.
+ */
+export const PRE_SANITIZED_AT = {
+	height: '2669784',
+	hash: kusamaRegistry.createType(
+		'BlockHash',
+		'0x5f2a8b33c24368148982c37aefe77d5724f5aca0bcae1a599e2a4634c1f0fab2'
+	),
+};
+
+/**
+ * A dummy return value to fetchStakingLedger which has not been run through `sanitizeNumbers`.
+ */
+export const PRE_SANITIZED_STAKING_RESPONSE = {
+	at: PRE_SANITIZED_AT,
+	staking: kusamaRegistry.createType('StakingLedger', {
+		stash: '5DRihWfVSmhbk25D4VRSjacZTtrnv8w8qnGttLmfro5MCPgm',
+		total: '0x0000000000000000ff49f24a6a9c00',
+		active: '0x0000000000000000ff49f24a6a9100',
+		unlocking: [],
```

Comment:
Is this a real example? Normally `total = active + unlocking.sum()`. It should work because that shouldn't affect `sanitizeNumbers`, but I don't think it's an actual state.

Reply:
Also why are these 15 bytes? Should be 16, no?

Reply:
Let's work on this in follow ups, but shouldn't block this.